tests/test_standard_metadata.py

```diff
@@ -1444,8 +1444,8 @@ def test_statically_defined_dynamic_field() -> None:
     "value",
     [
         "<3.10",
-        ">3.7,<3.11",
-        ">3.7,<3.11,!=3.8.4",
+        ">3.8,<3.11",
+        ">3.8,<3.11,!=3.8.4",
```

Comment:
I don't think this was necessary 🙂

Reply:
No, but probably doesn't hurt either. Best to forget 3.7 existed. :)